crawl-ref/source/dat/species/spriggan.yaml

```diff
@@ -37,7 +37,7 @@ aptitudes:
 size: little
 str: 4
 int: 9
-dex: 11
+dex: 10
```

Comment:
total is now not 24; bump int?

Reply:
Why should the total be 24?

Reply:
i thought most races were 24, based on the human 8 8 8

Reply:
Seems possible, but not very important if so. Not like races are so similar or so perfectly balanced that equal stat totals are meaningful.